src/util/parseCsv.ts

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const papaParser = require('papaparse');
+
+export function parseCSVToJSON(csvPath: string | File) {
```

Comment:
@ajaykrishnareddy-n please add a comment over here on why it was necessary to have type as `File` for node runtime

Reply:
For time being, changed the `string` type to `File | string` to satisfy the typescript. In the future if webpack provides approach or better solution will update it

Reply:
I meant in the code